set.mm

```diff
@@ -45957,61 +45982,78 @@ This theorem is proved directly from set theory axioms (no set theory
   ${
     snopeqop.a $e |- A e. _V $.
     snopeqop.b $e |- B e. _V $.
-    snopeqop.c $e |- C e. _V $.
-    snopeqop.d $e |- D e. _V $.
     $( Equivalence for an ordered pair equal to a singleton of an ordered pair.
-       (Contributed by AV, 18-Sep-2020.) $)
+       (Contributed by AV, 18-Sep-2020.)  (Revised by AV, 15-Jul-2022.)  (Avoid
+       depending on this detail.) $)
     snopeqop $p |- ( { <. A , B >. } = <. C , D >.
                      <-> ( A = B /\ C = D /\ C = { A } ) ) $=
-      ( wceq cop csn wa w3a opeqsn anbi2i eqcom opex bitri 3anan12 3bitr4i ) CD
-      IZABJZCKIZLZUAABIZCAKIZLZLUBKZCDJZIZUEUAUFMUCUGUAABCEFGNOUJUIUHIUDUHUIPCD
-      UBGHABQNRUEUAUFST $.
+      ( cvv wcel cop csn wceq wb wa a1i wn wi c0 eqeq2d syl6bi eleq1 w3a ancoms
+      eqcom opeqsng syl5bb opeqsni anbi2d 3anan12 bicomi 3bitrd opprc2 wne opex
+      snnz eqneqall mpi adantr notbid eqcoms pm2.21 impd com12 impbid pm2.61ian
+      3ad2ant2 opprc1 snex pm2.24i 3ad2ant3 pm2.61i ) CGHZABIZJZCDIZKZABKZCDKZC
+      AJZKZUAZLZDGHZVKWAWBVKMZVOVQVLCJKZMZVQVPVSMZMZVTVOVNVMKZWCWEVMVNUCVKWBWHW
+      ELCDVLGGUDUBUEWCWDWFVQWDWFLWCABCEFUFNUGWGVTLWCVTWGVPVQVSUHUINUJWBOZVKMZVO
+      VTWIVOVTPVKWIVOVMQKZVTWIVNQVMCDUKRWKVMQULVTVLABUMUNVTVMQUOUPZSUQVTWJVOVQV
+      PWJVOPVSVQWIVKVOVQWIVKOZVKVOPWIWMLDCDCKWBVKDCGTURUSVKVOUTSVAVEVBVCVDWMVOV
+      TWMVOWKVTWMVNQVMCDVFRWLSVTWMVOVSVPWMVOPVQVSWMVRGHZOVOVSVKWNCVRGTURWNVOAVG
+      VHSVIVBVCVJ $.
```

Comment:
I'm surprised that the proof is that much longer ? I guess I'll wait for the webpages to have a look.

Reply:
The cases ` C e/ _V ` and ` D e/ _V ` must be handled separately which makes the proof longer. Then we can assume ` C e. _V ` and ` D e. _V ` for the rest of the proof, and therefore the corresponding two hypotheses can be removed. This is also the reason why ~opeqsng ist needed.